mlir/lib/Catalyst/Transforms/BufferizationPatterns.cpp

```diff
@@ -110,8 +110,47 @@ struct BufferizePythonCallOp : public OpConversionPattern<PythonCallOp> {
             bufferArgs.push_back(newBuffer);
         }
```

Comment:
An inactive callback has no results right?

Reply:
Ah maybe I got this wrong because I though debug.callback -> inactive_callback, but I guess the active_callback also calls the inactive callback.

Reply:
I am not sure if you were saying that I should get rid of the results in the InactiveCallbackOp. I just did. The reason why I didn't get rid of it is because I was still exploring different ways to implement this. To be honest, I have a better way in mind now :sweat_smile: but I will prioritize getting this PR in and subsequent PRs will focus on cleaning up.

Reply:
I agree with you, but it is difficult to reconcile the "please close tickets before the end of iteration" with "no rough first implementation PRs" when:

1. we did not have time to design the whole epic in advance to forsee everything in each story
2. stories build upon previous stories that may not be used until later.

I do now know if it would be possible to have tickets that do not correspond to merging of PRs but instead to just get the implementation. And a final ticket for merging the implementation.

That to me, seems appropriate for large features like this one.